python/ray/actor.py

```diff
@@ -285,6 +286,7 @@ class ActorClassMetadata:
         accelerator_type: The specified type of accelerator required for the
             node on which this actor runs.
         runtime_env: The runtime environment for this actor.
+        scheduling_strategy: Strategy about how to schedule this actor.
```

Comment:
This means we allow to create
```
@ray.remote(scheduling_strategy="blah blah")
def f():
    
```

right?

Reply:
Hmm can we only disallow this to the placement group? I think it makes no sense to set this strategy for the placement group.

Reply:
@ericl, Sang's suggesting that PlacementGroupSchedulingStrategy (not other SchedulingStrategy) can only be set through `Options` not decorator.

So @ray.remote(scheduling_strategy=PlacementGroupSchedulingStrategy(...)) is not allowed.

WDYT?

Reply:
I don't see a strong reason to disallow it. You might create a placement group in a function and define nested remote functions closing over the placement group.

It doesn't cause any issues not already present in placement groups, and would be a special case to disallow, so I vote to allow it (default decision for all args).

Reply:
Got it. It sounds reasonable. I think in this case, I'd like to at least make sure the documentation states its behavior clearly. 

@jjyao can you update the documentation to make it clear this won't work if placement group is removed?

Reply:
I feel a better place to state the behavior is PG doc not the code comment here.